### DISCUSSION TITLE
No entities?

### DISCUSSION THREAD

**Discussion by dan-lindley** [blakeblackshear/frigate#6261] [Ask A Question] [answered]
Firstly thank you to Nick-M for helping me get all set up with frigate. It's working fairly well. 

I wanted to set up cards for the wife to use on her dashboard. I have installed the frigate card integration via HACs and that seems to be running ok, the only issue i have is that frigate doesn't appear to have created any entities for my cameras?

I have MQTT (mosquito) installed and zigbee2mqtt is working fine with the same credentials and settings. There are no MQTT errors on the logs but there are no entities being created. Iv e done a couple of full restarts during this process and a reinstall of the frigate integration and still can't see the camera feeds as entities. I wonder if i am missing something? Ive had a search and struggle to find the answer.

Any help would be appreciated. 

Dan

**Comment by NickM-27** [comment, marked as answer]
Seems like you haven't installed the frigate integration

**Reply by dan-lindley** [reply]
You helped me set it up yesterday! :) i have frigate installed and have done a reinstall.

**Reply by NickM-27** [reply]
Frigate and the integration are different things

**Reply by dan-lindley** [reply]
Just seen this in HACS thats definitely a face palm moment, will let you know how i get on.

**Comment by dan-lindley** [comment]
is the HACS integration not for those running frigate separately ie: outside of HA. I'm running it as an addon?

**Reply by NickM-27** [reply]
Please read the docs https://docs.frigate.video/frigate/installation

> Frigate is a Docker container that can be run on any Docker host including as a HassOS Addon. Note that a Home Assistant Addon is not the same thing as the integration. The integration is required to integrate Frigate into Home Assistant.

**Reply by dan-lindley** [reply]
Aaah right, thats confusing, just doing a reboot now to see if that has helped

**Reply by NickM-27** [reply]
There's a lot of different applications that have an addon and an integration, it's important to know the difference between them 👍

**Reply by dan-lindley** [reply]
Thank you. again you've been a great help. Still doesn't want to work. Have the HACs integration installed and the add-on. 
No entities appear at all in HA for anything other than the frigate update.

**Reply by NickM-27** [reply]
Did you setup the integration after installing it?

**Comment by dan-lindley** [comment]
apparently i did not! :/ this has been a complex journey! Thank you again, you will be pleased to know too that the URL would not connect but i managed to find that it was the default and i needed the full access url. ::phew::
thats 2 beers now! :)
